XLS-0065-single-asset-vault/README.md

```diff
@@ -613,7 +613,9 @@ In sections below assume the following variables:
     - The shares `MPToken.MPTAmount` of the `Account` is less than $\Delta_{share}$ (attempt to withdraw more shares than owned).
     - `Vault.AssetsAvailable` < `Amount` (the vault has insufficient assets).
 
-- The `Destination` account is specified and it does not have permission to receive the asset.
+- The `Destination` account is specified:
+  - The account does not have permission to receive the asset.
+  - The account does not have a `RippleState` or `MPToken` object for the asset.
```

Comment:
Would it be worth adding a case for "The account does not exist", or is it sufficient that it be implied by the other two checks. i.e. If the account doesn't exist, it certainly won't have a line or token.